tensorflow/core/kernels/mkl_tfconv_op.cc

```diff
@@ -29,8 +29,8 @@ limitations under the License.
 #include "tensorflow/core/util/tensor_format.h"
 
 #include "tensorflow/core/util/mkl_util.h"
-#include "third_party/mkl/include/mkl_dnn.h"
-#include "third_party/mkl/include/mkl_dnn_types.h"
+#include "mkl_dnn.h"
```

Comment:
Could you elaborate on 'various issues'?
It worked as is before, because of all the non-standard things that were done in configure. This is what we do for any other external dependency headers, libjpeg, libpng, etc. The path here was just a hand fabricated path due to the way things were copied in `confiure`